apps/docs/pages/guides/platform/database-usage.mdx

```diff
@@ -37,6 +42,13 @@ If you need more than 1024TB of database storage or require multiple storage exp
 
    This allows you to delete data from within the session.
 
+### Preoccupied Space
+
+When launching a new project, several extensions will be installed to your project.
+This uses ~40-60mb and will be included in your database size.
```

Comment:
To clarify, preinstalled extensions use up ~40-60MB of a project's allotted db size? Any detriments if any of the preinstalled extensions are disabled?

Reply:
I wouldn't mention/recommend it, as this could cause issues with their instance, just to save 1mb or so.